src/components/ebay-notice/README.md

```diff
@@ -19,3 +23,13 @@ Name | Type | Stateful | Description
 `status`  | String | No | "priority" (default), "confirmation" or "information"
 `aria-text` | String | No | adding description for the notice for a11y users
 `heading-level` | String | No| used in case of "page" level notices to specify the heading tag according to the notice's placement
+`dismissible` | Boolean | No | used to specify the dismiss button in page notice
+`hidden` | Boolean | Yes | whether the widget is hidden or not (Note: not supported as initial attribute)
+`aria-label-close` | String | No | adding description for the button for a11y users
+
+### ebay-notice Events
+
+Event | Data | Description
+--- | --- | ---
+`notice-show` | `{ hidden }` | the page notice was shown
+`notice-hide` | `{ hidden }` | the page notice was hidden
```

Comment:
I don't think the events need data anymore.

Reply:
done